src/handlers/no_merges.rs

```diff
@@ -102,11 +104,25 @@ pub(super) async fn handle_input(
         $ # delete any merge commits in the editor that appears
         $ git push --force-with-lease
         ```
+        "
+        .to_string()
+    };
 
+    let since_last_posted = if state.data.mentioned_merge_commits.is_empty() {
+        ""
+    } else {
+        " (since this message was last posted)"
+    };
+    write!(
+        message,
```

Comment:
A lot of these strings have a lot of whitespace indentation, which ends up causing them to be rendered as code blocks, such as:

        The following commits are merge commits:

Can the strings here use `\n\` at the end of lines to remove that whitespace so that it is better formatted as markdown?

Reply:
I don't have a strong preference. Another option is to move larger strings into a const like the [assign](https://github.com/rust-lang/triagebot/blob/0cfa95a8f7382d275a536da978f24f8be14d144f/src/handlers/assign.rs#L41-L69) module. You still need to be careful about newlines since GitHub markdown is newline-sensitive.